google/cloud/bigtable/test_proxy/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A minimal CMakeList.txt showing how to use the Cloud Bigtable test proxy
```

Comment:
This comment is not applicable.

Reply:
I think you used bigtable/quickstart/CMakeLists.txt as inspiration.  That is intended as a standalone file, used outside the larger project, and not really a good start.  Something like bigtable/benchmarks/CMakeLists.txt may be a better start.

Before you invest too much time into learning the ins and outs of supporting our CMake files (and then never using that knowledge ever again).  Talk to @dbolduc and figure out if he can help you.